core/test/xios_tests.cfg

```diff
@@ -0,0 +1,10 @@
+[model]
+start = 2023-03-17T17:11:00Z
+stop = 2023-03-17T23:11:00Z
+time_step = P0-0T01:30:00
+
+[ConfigOutput]
```

Comment:
Don't use the ConfigOutput section to configure XIOS output.

Reply:
Addressed in 89e37647dc13cbc9d02ba86620bb4841cb51d880.